src/collapse/mod.rs

```diff
@@ -253,7 +253,7 @@ impl Input {
         })
     }
 
-    fn chunks<'a>(&'a self) -> Chunks<'a> {
+    fn chunks(&self) -> Chunks<'_> {
```

Comment:
I don't think you need the `<'_>` here at all actually?

Reply:
Yep, that's right. Will change it to `Chunks`. I can never keep these elision rules 100% straight :).